src/cli/list.rs

```diff
@@ -21,7 +21,7 @@ use crate::Project;
 pub enum SortBy {
     Size,
     Name,
-    Type,
+    Kind,
```

Comment:
This is unfortunately a breaking change i would like to avoid. Can we add something like:
```suggestion
   #[value(alias("foo"))]
    Kind,
```

or 
```suggestion
   #[serde(alias = "type")]
    Kind,
```